turbinia/config/__init__.py

```diff
@@ -68,7 +68,10 @@
     'API_SERVER_PORT',
     'API_ALLOWED_ORIGINS',
     'API_AUTHENTICATION_ENABLED',
-    'WEBUI_PATH',
+    'CHUNK_SIZE',
```

Comment:
prefix the config variable name with API_

API_UPLOAD_CHUNK_SIZE
API_EVIDENCE_UPLOAD_DIR 

etc